### ISSUE TITLE
[Feature Request] Setting for selecting parent folder depth

### ISSUE THREAD

**Issue by yjagota** [oliverschwendener/ueli#123]
Hello,

Please add a setting for selecting parent folder depth in file/folder search so that description shows 'n' number of parent folders for the file/folder.

For e.g.: If I have following files in C:\SearchFolder which is added in File search options ->_ Folder path:
1. C:\SearchFolder\abc\somefolder\file1.txt
2. C:\SearchFolder\abc\somefolder\file2.txt
3. C:\SearchFolder\xyz\somefolder\file1.txt
4. C:\SearchFolder\xyz\somefolder\file2.txt
... and I search file1, search will show the following results:
file1.txt
somefolder > file1.txt
file1.txt
somefolder > file1.txt

It is difficult to understand which file is which in this case.

A parent folder depth of 2 will display the following results:
file1.txt
abc > somefolder > file1.txt
file1.txt
xyz > somefolder > file1.txt

Thanks & Regards.

**Comment by oliverschwendener** [comment]
Good idea! Will look into this as soon as possible!

**Comment by yjagota** [comment]
Great. Thanks.

BTW, displaying the parent folder is also not a bad idea (and a setting for the same):
file1.txt
C:\SearchFolder\abc\somefolder > file1.txt
file1.txt
C:\SearchFolder\xyz\somefolder > file1.txt

This will save a lot of parsing time/overhead.

**Comment by oliverschwendener** [comment]
Maybe just an option like "show full file path". If enabled it will show this:

```
C:\SearchFolder\abc\somefolder\file1.txt
```

disabled it will show this:

```
somefolder > file1.txt
```

**Comment by oliverschwendener** [comment]
This is included in the newest [v8.0.0](https://github.com/oliverschwendener/ueli/releases/v8.0.0) release.
